Packs/Polygon/Integrations/Polygon/Polygon.py

```diff
@@ -377,25 +411,51 @@ def get_monitor_indicators(report):
             end_time=p.get('exited_at'),
             path=p.get('filename'),
         )
-        ids.append(process)
+        command_results.append(CommandResults(
+            readable_output=get_human_readable_analysis_info(analysis_info),
+            outputs_prefix="Polygon.Analysis",
+            outputs_key_field="ID",
+            outputs=analysis_info,
+            indicator=process,
+            raw_response=res
+            )
+        )
         for regkey in p.get('regkeys', []):
             if regkey.get('action') == 'regkey_written':
                 reg = RegistryKey(
                     path=regkey.get('ioc'),
                     value=str(regkey.get('value'))
                 )
-                ids.append(reg)
-
-    return ids
-
-
-def get_report_indicators(report, analysis_type):
-    indicators = [get_main_indicator(report, analysis_type)]
-    indicators += get_packages_indicators(report)
-    indicators += get_network_indicators(report)
-    indicators += get_monitor_indicators(report)
+                command_results.append(CommandResults(
+                    readable_output=get_human_readable_analysis_info(analysis_info),
+                    outputs_prefix="Polygon.Analysis",
+                    outputs_key_field="ID",
+                    outputs=analysis_info,
+                    indicator=reg,
+                    raw_response=res
```

Comment:
Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.